apps/emqx_authn/test/emqx_authn_mongo_tls_SUITE.erl

```diff
@@ -169,15 +169,15 @@ raw_mongo_auth_config(SpecificSSLOpts) ->
       topology => #{
                     server_selection_timeout_ms => <<"10000ms">>
                    },
-      
+
       ssl => maps:merge(SSLOpts, SpecificSSLOpts)
      }.
 
 mongo_server() ->
     iolist_to_binary(
       io_lib:format(
-        "~s:~b",
-        [?MONGO_HOST, ?MONGO_PORT])).
+        "~s",
+        [?MONGO_HOST])).
```

Comment:
I don't like such code style as it puts only one parameter in one line, which increase total line of the source file.

Reply:
Didn't think so much. 
Will be changed to a single line in conflict resolution.
(Caused by _"`on_jsonify/1` removed"_ merged)